src/time_integration/paired_explicit_runge_kutta/polynomial_optimizer.jl

```diff
@@ -0,0 +1,138 @@
+using LinearAlgebra: eigvals
+using ECOS
+using Convex
+const MOI = Convex.MOI
+
+function read_in_eig_vals(path_to_eval_file)
+    # Declare and set to some value
+    num_eig_vals = -1
+    open(path_to_eval_file, "r") do eval_file
+        num_eig_vals = countlines(eval_file)
+    end
+    eig_vals = Array{Complex{Float64}}(undef, num_eig_vals)
+    line_index = 0
+    open(path_to_eval_file, "r") do eval_file
+        # Read till end of file
+        while !eof(eval_file)
+            # Read a new / next line for every iteration          
+            line_content = readline(eval_file)
+            eig_vals[line_index + 1] = parse(Complex{Float64}, line_content)
+            line_index += 1
+        end
+    end
+
+    return num_eig_vals, eig_vals
+end
```

Comment:
We can probably remove this, but we should double-check.

Reply:
I deleted it and ran it with the very old version of the elixir that downloaded the coefficient file. Works fine!